src/components/Map/__tests__/Map.spec.tsx

```diff
@@ -23,84 +26,131 @@ describe('<Map/> that renders a Map with markers', () => {
             lng: 4.9338793,
         },
         radius: 30000,
-    };
+    } as CircularMarker;
     const pointMarker = {
         center: {
             lat: 52.5112671,
             lng: 7.2535521,
         },
-    };
+    } as CircularMarker;
 
     it('should render with default props', () => {
-        const wrapper = mount(<Map />);
-        expect(toJson(wrapper)).toMatchSnapshot();
+        const view = render(<Map />);
+
+        expect(view.container).toMatchSnapshot();
     });
+
     it('should set center and zoom when rendered with default props', () => {
-        mount(<Map />);
+        render(<Map />);
+
         expect(setZoomMock).toHaveBeenCalledTimes(1);
         expect(setCenterMock).toHaveBeenCalledTimes(1);
     });
+
     it('should fit bounds when rendered with defaultArea as address', () => {
         geocodeMock.mockImplementationOnce((req, cb) => {
             cb(geocodeResponse.results, geocodeResponse.status);
         });
-        mount(<Map defaultArea={{ address: 'nl' }} />);
+        render(<Map defaultArea={{ address: 'nl' }} />);
+
         expect(geocodeMock).toHaveBeenCalledTimes(1);
         expect(fitBoundsMock).toHaveBeenCalledTimes(1);
     });
+
     it('should add default highlight', () => {
-        mount(<Map defaultHighlight={NL_PATHS} />);
+        render(<Map defaultHighlight={NL_PATHS as GeoJSON.GeoJsonObject} />);
+
         expect(addGeoJsonMock).toHaveBeenCalledTimes(1);
     });
+
     it('should remove default highlight when markers added', () => {
-        const wrapper = mount(<Map defaultHighlight={NL_PATHS} />);
+        const view = render(<Map defaultHighlight={NL_PATHS as GeoJSON.GeoJsonObject} />);
+
         expect(addGeoJsonMock).toHaveBeenCalledTimes(1);
 
-        wrapper.setProps({ markers: [pointMarker, regionMarker] });
+        view.rerender(
+            <Map
+                defaultHighlight={NL_PATHS as GeoJSON.GeoJsonObject}
+                markers={[pointMarker, regionMarker]}
+            />
+        );
         // will be called as many times as the length of the returned array from addGeoJsonMock
         expect(removeDataMock).toHaveBeenCalledTimes(1);
     });
+
     it('should render with markers', () => {
-        const wrapper = mount(<Map markers={[pointMarker, regionMarker]} />);
-        expect(wrapper.find('Marker')).toHaveLength(2);
-        expect(wrapper.find('Circle')).toHaveLength(1);
+        const view = render(<Map markers={[pointMarker, regionMarker]} />);
+
+        expect(view.container).toMatchSnapshot();
```

Comment:
Could you please switch to my branch and open MapWithGoogleLoader.spec.tsx just delete // @ts-ignore

Reply:
I've read some topics and what's the point to test something that doesn't show

Reply:
Indeed, it doesn't make sense to test things that do not affect the UI. However that is not the case here.Here it doesn't show only because we are mocking a very complex library. It would still be good in this case, to see that props are passed correctly, as there is quite a bit of logic around that in the code. Can you please research if that is possible?

Reply:
I've not found  solution. 
Only about enzyme:
https://stackoverflow.com/questions/54960750/test-that-react-prop-method-has-been-called-with-jest